stacks-signer/src/v0/signer.rs

```diff
@@ -588,21 +582,12 @@ impl Signer {
             }
             block_info.signed_self.get_or_insert(get_epoch_time_secs());
         }
-        // Record the block validation time
-        let non_bootcode_contract_call_block = block_info.block.txs.iter().any(|tx| {
-            // We only care about blocks that contain a non bootcode contract call
-            match &tx.payload {
-                TransactionPayload::ContractCall(cc) => !cc.address.is_boot_code_addr(),
-                TransactionPayload::SmartContract(..) => true,
-                _ => false,
-            }
-        });
-        if non_bootcode_contract_call_block {
-            block_info.validation_time_ms = Some(block_validate_ok.validation_time_ms);
+        // Record the block validation time but do not consider stx transfers or boot contract calls
+        if block_validate_ok.cost.is_zero() {
+            0
         } else {
-            // Ignore purely boot code and stx transfers when calculating the processing/validation time
-            block_info.validation_time_ms = Some(0);
-        }
+            block_validate_ok.validation_time_ms
+        };
```

Comment:
This isn't used anywhere. Is there something missing here?

Reply:
Yes indeed there is.

Reply:
Fixed!